Documentation/dnx-instructions.md

```diff
@@ -0,0 +1,179 @@
+	                "System.Console": "4.0.0-beta-*"
+	            }
+	        }
+	    }
+	}
+
+You need to restore any dependent nuget packages, with the dnu tool (comes with DNX). DNU will pull down NuGet packages from nuget.org or any other configured NuGet feed, such as MyGet.
+
+	C:\coreclr-demo>dnu restore
+
+You can now run the app, with dnx.
+
+	C:\coreclr-demo>dnx . run
+
+Switching to CoreCLR
+====================
+
+For the purpose of this demo, you need an x64 DNX hosted on (actually including) CoreCLR. The CoreCLR build only generates X64 artifacts currently. You can install the latest version of  an X64 CoreCLR DNX with DNVM. You'll notice that the install process will crossgen the managed assemblies for you. Performance!
+
+	C:\coreclr-demo>dnvm install latest -arch x64 -runtime coreclr
+
+You should now have two runtimes installed. The CoreCLR one will be the default. The install process calls `dnvm use` for you as a convenience, since that's probably what you wanted. `dnvm list` will demonstrate this change in state.
+
+	C:\Users\rlander>dnvm list
+
+	Active Version           Runtime Architecture Location                       Al
+	                                                                             ia
+	                                                                             s
+	------ -------           ------- ------------ --------                       --
+	       1.0.0-beta5-11447 clr     x86          C:\Users\rlander\.dnx\runtimes de
+	  *    1.0.0-beta5-11447 coreclr x64          C:\Users\rlander\.dnx\runtimes
+
+You will need to repeat the steps above to run your app on CoreCLR (CoreCLR requires more NuGet packages).
+
+	C:\coreclr-demo>dnu restore
+	C:\coreclr-demo>dnx . run
+
+Creating a Custom DNX
+=====================
+
+By this point, the demo is working with an official DNX. You can now create a custom DNX with a custom build of CoreCLR. For the purpose of safe experimentation, you are recommended to copy and rename an existing DNX. Note that the version numbers in the example below might not match your current environment. You'll need to adjust.
+
+	C:\coreclr-demo>dir %USERPROFILE%\.dnx\runtimes
+	 Volume in drive C has no label.
+	 Volume Serial Number is F074-BCDF
+
+	 Directory of C:\Users\rlander\.dnx\runtimes
+
+	04/01/2015  04:45 PM    <DIR>          .
+	04/01/2015  04:45 PM    <DIR>          ..
+	04/01/2015  04:33 PM    <DIR>          dnx-clr-win-x86.1.0.0-beta5-11447
+	04/01/2015  04:45 PM    <DIR>          dnx-coreclr-win-x64.1.0.0-beta5-11447
+	               0 File(s)              0 bytes
+	               4 Dir(s)  94,895,509,504 bytes free
+
+
+Copy and rename the X64 CoreCLR DNX. That's the one you'll target. You'll be able to see it with `dnvm list`. This time, you will need to explictly `use` this DNX in order to use it.
+
+	C:\coreclr-demo>xcopy /E /S %USERPROFILE%\.dnx\runtimes\dnx-coreclr-win-x64.1.0.0-beta5-11469 %USERPROFILE%\.dnx\runtimes\dnx-coreclr-win-x64.1.0.0-beta5-11469c
```

Comment:
nit: it is quite hard to visually see the `c` (which I suppose stands for `custom`?), can we just append `custom` here?